Email_Templates/Credentials/SurveyHound/email.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+	<body style="background:#1C2127; height:700px;">
+		<table style="width:900px; border:1px solid #666; border-radius:5px; background:#DDD; margin:0 auto; text-align:center; height:500px;">
+			<th>{{ inline_image('/media/shadowman/Secondary/Projects/SME/surveyhound-header.png') }}</th>
```

Comment:
Please provide the inline image being referenced here. Also change your inline_image path to something to more repersent where the image will be located in the repository:
{{ inline_image('/king-phisher-templates/Email_Templates/Credentials/SurveyHound') }}

also please prove a README.md file for this template folder on recommend or required changes for the email template.